src/Confix.Tool/src/Confix.Tool/Variables/LocalVariableProviderConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+using System.Text.Json.Nodes;
+
+namespace ConfiX.Variables;
+
+public record LocalVariableProviderConfiguration
```

Comment:
Should we aim to use records for all configurations? I was considering it, but I wasn't fully convinced. Primary constructors are expected to be introduced in C# 12. So, perhaps we could use records first and then class/structs with primary constructors later?